crypto/cmp/cmp_hdr.c

```diff
@@ -72,6 +72,16 @@ ASN1_OCTET_STRING *OSSL_CMP_HDR_get0_recipNonce(const OSSL_CMP_PKIHEADER *hdr)
     return hdr->recipNonce;
 }
 
+STACK_OF(OSSL_CMP_ITAV)
+    *OSSL_CMP_HDR_get0_geninfo_ITAVs(const OSSL_CMP_PKIHEADER *hdr)
+{
+    if (hdr == NULL) {
```

Comment:
Inconsistency with other functions that require an argument (e.g. most `SSL_` functions).

Reply:
I guess you mean the NULL check for the parameter.
This style is consistent with all the CMP API functions.
When I contributed them, I was told that all parameters of API functions shall be checked and to add an error queue entry on failure.

Reply:
I think he means the function signature i.e.
int func(*obj, **ret)

Reply:
Probably a bit late now :)